From e717406eac1875f08092290ef35065365fcd1873 Mon Sep 17 00:00:00 2001
From: Ivan Shvedunov <ivan4th@gmail.com>
Date: Wed, 5 Aug 2020 16:44:28 +0300
Subject: [PATCH] Increase reassembly limit

This is an interim solution till we implement a proper CLI command
---
 src/vnet/ip/reass/ip4_full_reass.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/vnet/ip/reass/ip4_full_reass.c b/src/vnet/ip/reass/ip4_full_reass.c
index 5b69234e4..8e961e2f3 100644
--- a/src/vnet/ip/reass/ip4_full_reass.c
+++ b/src/vnet/ip/reass/ip4_full_reass.c
@@ -37,7 +37,7 @@
  * running out context space and unable to perform reassembly */
 #define IP4_REASS_EXPIRE_WALK_INTERVAL_DEFAULT_MS 50 // 50 ms default
 #define IP4_REASS_MAX_REASSEMBLIES_DEFAULT 1024
-#define IP4_REASS_MAX_REASSEMBLY_LENGTH_DEFAULT	  3
+#define IP4_REASS_MAX_REASSEMBLY_LENGTH_DEFAULT 8
 #define IP4_REASS_HT_LOAD_FACTOR (0.75)
 
 #define IP4_REASS_DEBUG_BUFFERS 0
-- 
2.30.2

